chalicelib_smaht/check_setup.json

```diff
@@ -42,259 +42,206 @@
         "schedule": {
             "fifteen_min_checks_3": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true,
                         "queue_action": "prod"
                     },
                     "dependencies": []
                 }
             }
-        }
+        },
+        "display": [
+            "staging"
+        ]
     },
     "spot_failed_metawfrs": {
         "title": "Spot-failed MetaWorkflowRuns",
         "group": "Pipeline Checks",
         "schedule": {
             "fifteen_min_checks_2": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true,
                         "queue_action": "prod"
                     },
                     "dependencies": []
                 }
             }
-        }
+        },
+        "display": [
+            "staging"
+        ]
     },
     "failed_metawfrs": {
         "title": "Failed MetaWorkflowRuns",
         "group": "Pipeline Checks",
         "schedule": {
             "manual_checks": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true
                     }
                 }
             }
         },
         "display": [
-            "<env-name>"
+            "staging"
         ]
     },
     "find_meta_workflow_runs_requiring_output_linktos": {
         "title": "MetaWorkflowRuns to PATCH output files",
         "group": "Pipeline Checks",
         "schedule": {
             "hourly_checks_2": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true,
                         "queue_action": "prod"
                     }
                 }
             }
-        }
+        },
+        "display": [
+            "staging"
+        ]
     },
     "find_meta_workflow_runs_with_linkto_errors": {
         "title": "MetaWorkflowRuns with output file PATCH errors",
         "group": "Pipeline Checks",
         "schedule": {
             "manual_checks": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true
                     }
                 }
             }
         },
         "display": [
-            "<env-name>"
+            "staging"
         ]
     },
     "find_meta_workflow_runs_to_kill": {
         "title": "MetaWorkflowRuns to stop",
         "group": "Pipeline Checks",
         "schedule": {
             "manual_checks": {
-                "<env-name>": {
+                "data": {
                     "kwargs": {
                         "primary": true
                     }
                 }
             }
         },
         "display": [
-            "<env-name>"
-        ]
-    },
-    "find_sample_processing_for_meta_workflow": {
```

Comment:
If this is going to come back when David implements metaworkflows, you could implement a convention with a leading underscore that makes it be disabled but still present. Since I assume comments don't work here. But this is fine I guess.